keras_preprocessing/image/directory_iterator.py

```diff
@@ -140,9 +140,13 @@ def __init__(self,
                                                 shuffle,
                                                 seed)
 
+        self.path_to_files = [
+            os.path.join(self.directory, fname) for fname in self.filenames
+        ]
+
     @property
     def filepaths(self):
-        return [os.path.join(self.directory, fname) for fname in self.filenames]
+        return self.path_to_files
```

Comment:
Can you call the attribute `self.filepaths`? It is a common practice to add a prefix underscore if the attribute will be exposed via a property, thanks. 
Perhaps we do not need to create that intermediate `root` variable.
and can you change this logic above the instantiation of the parent class? Line 138

Reply:
Yes, I fixed my code according to your remark